packages/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -2145,6 +2145,8 @@
 	"workflowPreview.executionMode.showError.previewError.message": "Unable to preview workflow execution",
 	"workflowPreview.showError.previewError.title": "Preview error",
 	"workflowRun.noActiveConnectionToTheServer": "Lost connection to the server",
+	"workflowRun.showError.deactivate": "Deactivate workflow to execute",
+	"workflowRun.showError.prodactionActive": "Because of limitations in {nodeName}, n8n can't listen for test executions at the same time as listening for production ones",
```

Comment:
```suggestion
	"workflowRun.showError.productionActive": "Because of limitations in {nodeName}, n8n can't listen for test executions at the same time as listening for production ones",
```